tests/finite_element_problem/test_log_levels.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+from fenicsxconcrete.experimental_setup.cantilever_beam import CantileverBeam
+from fenicsxconcrete.finite_element_problem.linear_elasticity import LinearElasticity
+from fenicsxconcrete.unit_registry import ureg
+
+
+
+@pytest.mark.parametrize("log_level", [['DEBUG',False],
+                                       ['INFO',False],
+                                       ['ERROR',False],
+                                       ['WARNING',False],
+                                       ['CRITICAL',False],
+                                       ['some_string_that is_not_implemented',True]
+                                       ])
+def test_log_levels(log_level):
+    """This function tests all implemented log level in the base material init"""
+
+    log_str = log_level[0]
+    error_flag = log_level[1]
+
+    default_experiment, fem_parameters = LinearElasticity.default_parameters()
+    fem_parameters['log_level'] = log_str * ureg('')
```

Comment:
would it make sense to have a method `set_log_level` for the base material?
I guess people might also expect to set the log level via integer values? (which does not seem to be possible?)

Reply:
this is also related to #45 or rather the discussion on what kind of logging should be used and then also how the user would control that.
set the log level for all classes at once or specify different log levels for experiment or material classes.

Reply:
I agree, this should be dealt with in #45, this current implementation is be no means finished, optimized or ideal.
I think a single log level for everything would be best.